docs/ROADMAP.md

```diff
@@ -91,28 +91,28 @@ New stuff - curl
    of course.
 
 2. Simplify the SMTP command line so that the headers and multi-part content
-   don't have to be constructed before calling curl
+   don't have to be constructed before calling curl.
 
 Improve
 -------
 
-1. build for windows (considered hard by many users)
+1. Build for windows (considered hard by many users).
 
-2. curl -h output (considered overwhelming to users)
+2. Curl -h output (considered overwhelming to users).
 
-3. we have > 200 command line options, is there a way to redo things to
+3. We have > 200 command line options, is there a way to redo things to
    simplify or improve the situation as we are likely to keep adding
-   features/options in the future too
+   features/options in the future too.
 
-4. authentication framework (consider merging HTTP and SASL authentication to
-   give one API for protocols to call)
+4. Authentication framework (consider merging HTTP and SASL authentication to
+   give one API for protocols to call).
 
 5. Perform some of the clean up from the TODO document, removing old
    definitions and such like that are currently earmarked to be removed years
-   ago
+   ago.
 
 Remove
 ------
 
-1. makefile.vc files as there is no point in maintaining two sets of Windows
-   makefiles. Note: These are currently being used by the Windows autobuilds
+1. Makefile.vc files as there is no point in maintaining two sets of Windows
```

Comment:
This is a file name and they are often case sensitive so this cannot be capitalized without changing meaning.

Reply:
fixed.